### ISSUE TITLE
Mirroring more than one channel

### ISSUE THREAD

**Issue by elsehow** [mafintosh/hyperirc#3]
When I try to mirror more than one channel at once, I get

``` sh
events.js:160
      throw er; // Unhandled 'error' event
      ^
OpenError: IO error: lock hyperirc.db/LOCK: Resource temporarily unavailable
    at /usr/local/lib/node_modules/hyperirc/node_modules/level-packager/node_modules/levelup/lib/levelup.js:119:34
    at /usr/local/lib/node_modules/hyperirc/node_modules/leveldown/node_modules/abstract-leveldown/abstract-leveldown.js:39:16
```

Perhaps level separate dbs could be named according to the freenode channel name?

In any case, love this project.

**Comment by mafintosh** [comment]
ah yes good point. we could use multilevel to fix this also

**Comment by elsehow** [comment]
shall i issue a pr?

**Comment by mafintosh** [comment]
yes please. you can use multileveldown so we get free failover

**Comment by rahatarmanahmed** [comment]
If we do this we ought to save `hyperirc.db` in a more permanent location. Right now it saves it in the cwd, which feels like littering. Maybe load/save `~/.hyperirc.db` by default, and have a flag to use a different file?

**Comment by elsehow** [comment]
Original issue fixed in 69ceabd9f53c3b560c90f9553fbbd18531efe0f6
